pkgs/development/tools/git-ps-rs/default.nix

```diff
@@ -21,7 +21,7 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-4wSm3H+98ZJZ+fZdLYshPKafRkPq98Pv3Lwh9o0be6U=";
```

Comment:
The hash is unchanged?

Reply:
Yeah, not too sure why this doesn't error.

Reply:
That's how fixed output derivations work, if you already have the store path of given hash in your store, no build is ever attempted.